app.rb

```diff
@@ -1,3 +1,7 @@
+require 'json'
+
+# rubocop:disable Metrics/ModuleLength
```

Comment:
Data should be stored in a separate JSON file. For example, rental data should be stored in `rentals.json`, and people data should be stored in `people.json`. Data is not being preserved anywhere when the user exits from the app.

Reply:
Thank you for letting us know where was the problem. We fixed everything and now data are stored in separate JSON files.

Reply:
Well addressed this!

Reply:
@DeliceLydia  Thank you a lot!